### DISCUSSION TITLE
Support of multiple Charge

### DISCUSSION THREAD

**Discussion by lieglw** [PatrikTrestik/homeassistant-solax-http#26] [Ideas]
Hi, thanks a lot for this integration. Use it nearly every day and it is very stable. Just thinking of a second charger in parallel mode.
Would this work with your integration?

**Comment by PatrikTrestik** [comment]
It should work by default. You have to enter correct IP address and RegNo/Password for second charger and give it unique name.
If there is any error, please let me know. There should be no problem in API, but integration was never tested in this setup.
I believe that I can fix it in case of bug.

**Reply by lieglw** [reply]
Thank You

**Reply by lieglw** [reply]
Hey, have installed the 2. EV Charger and it works really well.
No issues up to now.
